app/views/shared/_reservations_table.html.haml

```diff
@@ -24,6 +24,10 @@
             = link_to "#{od.product.facility.abbreviation} / #{product_name}".html_safe, facility_instrument_path(od.product.facility, od.product)
             - if od.note.present?
               .order-detail-extra.order-detail-note= "Note: #{od.note}"
+            - if od.product.respond_to?(:online?)
+              - unless od.product.online?
+                = tooltip_icon "icon-warning-sign",
```

Comment:
I'm thinking we should make this red to make it stand out more. What do you think?